### PR TITLE
Feat(Query): Default Seccomp Profile Disabled for docker compose

### PR DESCRIPTION
**Pull request by cxAndreFelicidade**
**Proposed Changes**
- Query: Seccomp Disabled for docker compose

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/dockerCompose/seccomp_disabled/metadata.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "404fde2c-bc4b-4371-9747-7054132ac953",
+  "queryName": "Seccomp Disabled",
```
Comment (rafaela-soares) [inline assets/queries/dockerCompose/seccomp_disabled/metadata.json:3]:
```suggestion
  "queryName": "Default Seccomp Profile Disabled",
```

Please, update the query folder name

File: assets/queries/dockerCompose/seccomp_disabled/query.rego [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    sec_opt := sec_opt_array[a]
+    contains(sec_opt,"seccomp:unconfined")
+    
+	result := {
+		"documentId": sprintf("%s", [resource.id]),
+		"searchKey": sprintf("services.%s.security_opt",[name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "Seccomp default profile is not being disabled.",
+		"keyActualValue": "Seccomp default profile is disabled.",
+		"searchLine": common_lib.build_search_line(["services", name, "security_opt"], []),
```
Comment (rafaela-soares) [inline assets/queries/dockerCompose/seccomp_disabled/query.rego:18]:
```suggestion
        "searchLine": common_lib.build_search_line(["services", name, "security_opt", a], []),
```

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

**Review by rafaela-soares — APPROVED** [review]
LGTM 🚀

File: assets/queries/dockerCompose/default_seccomp_profile_disabled/test/positive_expected_result.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+[
+    {
+      "queryName": "Default Seccomp Profile Disabled",
+      "severity": "MEDIUM",
+      "line": 13,
+      "filename": "positive1.yaml"
```
Comment (joaorufi) [inline assets/queries/dockerCompose/default_seccomp_profile_disabled/test/positive_expected_result.json:6]:
```suggestion
      "fileName": "positive1.yaml"
```

File: assets/queries/dockerCompose/default_seccomp_profile_disabled/test/positive_expected_result.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    },
+    {
+      "queryName": "Default Seccomp Profile Disabled",
+      "severity": "MEDIUM",
+      "line": 10,
+      "filename": "positive2.yaml"
```
Comment (joaorufi) [inline assets/queries/dockerCompose/default_seccomp_profile_disabled/test/positive_expected_result.json:12]:
```suggestion
      "fileName": "positive2.yaml"
```

**Review by joaorufi — CHANGES_REQUESTED** [review]

File: assets/queries/dockerCompose/default_seccomp_profile_disabled/query.rego [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+	service_parameters := resource.services[name]
+    sec_opt_array := service_parameters.security_opt
+    sec_opt := sec_opt_array[a]
+    contains(sec_opt,"seccomp:unconfined")
+    
+	result := {
+		"documentId": sprintf("%s", [resource.id]),
+		"searchKey": sprintf("services.%s.security_opt",[name]),
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "Seccomp default profile is not being disabled.",
```
Comment (joaorufi) [inline assets/queries/dockerCompose/default_seccomp_profile_disabled/query.rego:16]:
```suggestion
		"keyExpectedValue": "Seccomp default profile to not be disabled.",
```

**Review by joaorufi — CHANGES_REQUESTED** [review]

**Pull request merged**